src/main/java/org/apache/flink/benchmark/full/PojoSerializationBenchmark.java

```diff
@@ -86,42 +86,69 @@ public static void main(String[] args)
     }
 
     @Benchmark
-    public byte[] writePojo() throws IOException {
-        return write(pojoSerializer, pojo);
+    @OperationsPerInvocation(INVOCATIONS)
+    public int writePojo() throws IOException {
+        stream.pruneBuffer();
+        for (int i = 0; i < INVOCATIONS; i++) {
```

Comment:
Could you motivate why do we need multiple invocations in a single benchmark? Because of potential costs of `stream.pruneBuffer();` or `pojoBuffer.reset();`? Are they measurable?

As it is, I would be a little be concerned what magic JIT can do after inlining `pojoSerializer.serialize(pojo, stream);`  and unrolling the loop. That might be desireable (vectorisation) but as we are invoking it over and over again with the same parameters, some other magic could yield false results.